pkgs/games/runescape3/default.nix

```diff
@@ -0,0 +1,109 @@
+{ stdenv, lib, buildFHSUserEnv, dpkg, glibc, gcc-unwrapped, autoPatchelfHook, fetchurl, wrapGAppsHook
+  , gnome2, xorg
+  , libSM, libXxf86vm, libX11, glib, pango, cairo, gtk2-x11, zlib, openssl
+  , libpulseaudio
+  , SDL2, xorg_sys_opengl, libGL
+}:
+let
+
+  version = "2.2.9";
+  pname = "runescape-launcher";
```

Comment:
Maybe we could rename the folder from `runescape3` to `runescape-launcher` if it makes more sense?

Reply:
sure, I actually don't care about how its named :sweat_smile: